pages/facts.html

```diff
@@ -4,9 +4,88 @@
+</nav>
+
 <body>
+  <section>
+  <h1> Everything You Need To Know About Bidets </h1>
+    <p> </p>
+  </section>
+
+
+
+  <section>
+    <details class="accordion" open>
+      <summary class="accordion-title">A Fairly Accurate  History of Bidets</summary>
+      <p class="accordion-body"><iframe src="https://www.youtube.com/embed/Zdq6-3Mvj78" title="A Fairly Accurate History of Bidets" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe></p>
+    </details>
+  </section>
+
+  <section>
+    <details class="accordion">
+      <summary class="accordion-title">Not all bidet's are created equal</summary>
+      <p class="accordion-body">Find the right fit for you. Each bidet can offer a variety of perks. Think about what you prioritize before making a decision. Do you want cold, hot, warm water? Do you want a fan to dry your bum? Perhaps you want music to play. There are truly so many different options -- you will look forward to sitting on the toilet for the rest of your life.. </p>
+    </details>
+  </section>
+
+  <section>
+    <details class="accordion">
+      <summary class="accordion-title">Don't you care about the enviroment?</summary>
+      <p class="accordion-body">We believe toilet paper is not only overratted, but a threat to our future. If everyone switched to a bidet we could save about over 15 million trees yearly. Make the Lorax proud. </p>
+    </details>
+  </section>
   
+  <section> 
+    <details class="accordion">
+      <summary class="accordion-title">Health Benefits for you and your loved ones </summary>
+      <ul class="accordion-body">
+        <li>Bidets Banish Bacteria. Besides being nicely alliterative, this is a super true statement. By flushing bacteria out and away from the most sensitive areas of your body, bidets work to prevent conditions like bacterial prostatitis, an inflammatory condition that affects the prostates of about 8% of U.S. men. Left-behind bacteria can also cause itching and discomfort, which are some of the most common symptoms that drive people into the offices of colorectal surgeons</li> 
+        <li>Bidets Help Prevent UTIs. If you’re a woman who is curious about the history of and current place for bidets, hear this: bidets can help prevent UTIs caused by E. coli bacteria, which is the culprit in about 90% of all UTI cases.</li>
+        <li> Bidets Decrease Discomfort. For women with scarring from perineal tears or episiotomies during childbirth, dry wiping can be an uncomfortable experience. Luckily, a bidet can help decrease this. By directing a comforting, warm stream of water at the affected areas, bidets cut down on irritation and friction for a more pleasant go.</li> 
+        
+      </ul>
+    </details>
+  </section>
+
+  <section>
+    <details class="accordion">
+      <summary class="accordion-title">Misconceptions DEBUNKED</summary>
+      <p class="accordion-body"><ul>
```

Comment:
In a bizarre turn of events, placing this `ul` inside this `p` makes the text invisible. I was able to get it to come back by removing the `p` and putting the class on the `ul` as you did above:

```html
<ul class="accordion-body">
```